fred2/missionsave.cpp

```diff
@@ -2997,7 +2997,7 @@ int CFred_mission_save::save_objects() {
 			if (shipp->flags[Ship::Ship_Flags::Scramble_messages])
 				fout(" \"scramble-messages\"");
 			if (!(objp->flags[Object::Object_Flags::Collides]))
-				fout(" \"no-collide\"");
+				fout(" \"no_collide\"");
```

Comment:
It it intentional that this uses the underscore but the string in `missionparse.cpp` uses a hyphen?